src/components/config_profile/include/config_profile/ini_file.h

```diff
@@ -87,13 +87,19 @@ extern "C" {
 /*
  * @brief Write usage instructions to the end of the file
  * @param
- *
+ * 
+ * \deprecated This method will be removed in the next major release
+ * See Profile::UpdateValues for the new ini parsing code
+ * 
  * @return NULL if file or desired entry not found, otherwise pointer to fname
  */
 extern char* ini_write_inst(const char* fname, uint8_t flag);
```

Comment:
```suggestion
DEPRECATED extern char* ini_write_inst(const char* fname, uint8_t flag);
```

Reply:
Same applies to other deprecated functions

Reply:
Ooh I thought this didn't work on C code. Added in https://github.com/smartdevicelink/sdl_core/pull/3688/commits/55c1173c058a08d2fada989db0de86102ae1b990

Also, the ini_file_test.cc file was printing many warnings so I disabled deprecated warnings for that file only in https://github.com/smartdevicelink/sdl_core/pull/3688/commits/59bcec714886c216d7679ff953bef5e94c0741b8, let me know if you don't like that idea. :)